chore: update readme (#344)
litmuschaos/litmus-helm: README.md

```bash
helm install chaos litmuschaos/litmus --namespace=litmus

## ARM support

> ## Litmus helm chart depends on bitnami/mongodb, which uses a mongodb image not supported on ARM.

Reviewer: ```suggestion
## Litmus helm chart depends on bitnami/mongodb, which uses a mongodb image that is not build for ARM arch.
## For more details, refer to the issue: https://github.com/bitnami/charts/issues/7305
```

> ## To install Litmus on an ARM-based server, use a custom MongoDB ARM image instead of the default.

Reviewer: ```suggestion
## To install Litmus on an ARM-based machine, you need to provide a custom MongoDB ARM image.
```
